api/ca.proto

```diff
@@ -55,3 +60,9 @@ message GetRootCACertificateRequest {}
 message GetRootCACertificateResponse {
 	bytes certificate = 1;
 }
+
+message GetUnlockKeyRequest {}
+
+message GetUnlockKeyResponse {
+	bytes unlock_key = 1;
```

Comment:
The key size is 256 bits of CSPRNG output intended for symmetric use, right? If so: that's sufficient forever, against both classical and quantum attackers.